053-SentinelAutomatedResponse/Coach/README.md

```diff
@@ -0,0 +1,68 @@
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+# What The Hack - Sentinel Automated Response - Coach Guide
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+## Introduction
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+Welcome to the coach's guide for the Sentinel Automated Response What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+**NOTE:** If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+
+## Coach's Guides
+
+This hack includes optional [lecture presentations](../Coach/Resources) that introduces key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge.</br>
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+- Challenge-01: **[Architecture, Agents, Data Connectors and Workbooks](Solution-01.md)**
+   - Understand the various architecture and decide on the appropriate design based on the requirements. Install the appropriate data connector to import Windows security events and validate Log Analytics data ingestion.
+- Challenge-02: **[Custom Queries & Watchlists](Solution-02.md)**
+   -  Build a custom analytics rule to show data ingested through the connector. Create a Watchlist and add data, verify the data is available in Log Analytics.  Change the table retention time to 7 days
+- Challenge-03: **[Automated Response](Solution-03.md)**
+   -  Bulid a custom rule that alerts when your user ID logs into a server. Use a playbook to automatically close the incident only if login occurred from a known IP address
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+
+## Prerequisites
```

Comment:
This section is intended to be titled "Coach Prerequisites" as these are the prerequisites for a Coach to complete before hosting a hack event.  Use 2 hashes for the title:
"##Coach Prerequisites"

Please re-insert the boilerplate text from the template here:

"This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.

The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams."

Finally, for the 3 bulleted pre-reqs you listed, these should be moved down further to the "Additional Coach Prerequisites" section below.